tests/wallet/inscribe.rs

```diff
@@ -247,3 +247,16 @@ fn inscribe_with_optional_satpoint_arg() {
   TestServer::spawn_with_args(&rpc_server, &[])
     .assert_response_regex(format!("/content/{inscription_id}",), ".*HELLOWORLD.*");
 }
+
+#[test]
+fn inscribe_with_fee_rate() {
+  let rpc_server = test_bitcoincore_rpc::spawn();
+  create_wallet(&rpc_server);
+  rpc_server.mine_blocks(1);
+
+  CommandBuilder::new("--index-sats wallet inscribe degenerate.png --fee-rate 2.0")
+    .write("degenerate.png", [1; 520])
+    .rpc_server(&rpc_server)
+    .stdout_regex("commit\t[[:xdigit:]]{64}\nreveal\t[[:xdigit:]]{64}\n")
+    .run();
```

Comment:
We should check that the transaction that's broadcast and mined has the correct fee rate.